crypto/des/rpc_enc.c

```diff
@@ -1,30 +0,0 @@
-/*
```

Comment:
Great minds think alike, I guess :)
(BTW, while I was looking at my old notes about crufty things in the tree, I ran into OPENSSL_indirect_call(), which appears to be undocumented and unused in the tree, but it has also been that way for a very long time, so I am more inclined to believe that I am misreading how it is used than that it is safe to remove.)